meson.build

```diff
@@ -404,14 +404,9 @@ conf_data.set10('C_MT32EMU', get_option('use_mt32emu'))
 conf_data.set10('C_TRACY', get_option('tracy'))
 conf_data.set10('C_FPU', true)
 conf_data.set10('C_FPU_X86', host_machine.cpu_family() in ['x86', 'x86_64'])
-
-if get_option('enable_debugger') != 'none'
-    conf_data.set10('C_DEBUG', true)
-endif
-
-if get_option('enable_debugger') == 'heavy'
-    conf_data.set10('C_HEAVY_DEBUG', true)
-endif
+conf_data.set10('C_DEBUGGER', get_option('enable_debugger') != 'none')
+conf_data.set10('C_HEAVY_DEBUG', get_option('enable_debugger') == 'heavy')
```

Comment:
Maybe a consistency change to `C_HEAVY_DEBUGGER`? Now you could think it's a heavy debug logging mode or something.

Reply:
Well, in some cases, it is. I need to do a more thorough examination, but I think at least some of the C_HEAVY_DEBUG-wrapped code is more thorough logging of some DOSBox internals. I'll take a look and either split it in another commit, or change it soon